test/specs/apexReplayDebugger.e2e.ts

```diff
@@ -0,0 +1,179 @@
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Execute Anonymous Apex with Currently Selected Text', 1);
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running Execute Anonymous Apex', fiveMinutes);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'Execute Anonymous Apex successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+    await utilities.pause(1);
+  });
+
+  step('SFDX: Get Apex Debug Logs', async () => {
+    // Run SFDX: Get Apex Debug Logs
+    const workbench = await browser.getWorkbench();
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Get Apex Debug Logs', 1);
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Getting Apex debug logs', fiveMinutes);
+
+    // Select a log file
+    const quickPicks = await prompt.getQuickPicks();
+    expect(quickPicks).not.toBeUndefined();
+    expect(quickPicks.length).toBeGreaterThanOrEqual(1);
+    await prompt.selectQuickPick('User User - Api');
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Get Apex Debug Logs', fiveMinutes);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Get Apex Debug Logs successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+  });
+
+  step('SFDX: Launch Apex Replay Debugger with Last Log File', async () => {
+    // Run SFDX: Launch Apex Replay Debugger with Last Log File
+    const workbench = await browser.getWorkbench();
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Launch Apex Replay Debugger with Last Log File', 1);
+
+    // Get open text editor
+    const editorView = await workbench.getEditorView();
+
+    // Get file path from open text editor
+    const activeTab = await editorView.getActiveTab();
+    expect(activeTab).not.toBe(undefined);
+    const title = await activeTab?.getTitle();
+    const logFilePath = path.join(path.delimiter, 'tools', 'debug', 'logs', title!).slice(1);
+    await prompt.setText(logFilePath);
+    await prompt.confirm();
+    await utilities.pause(1);
+
+    // Continue with the debug session
+    await browser.keys(['F5']);
+    await utilities.pause(1);
+    await browser.keys(['F5']);
+    await utilities.pause(1);
+  });
+
+  step('SFDX: Launch Apex Replay Debugger with Current File', async () => {
+    // Run SFDX: Launch Apex Replay Debugger with Current File
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Launch Apex Replay Debugger with Current File', 1);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Launch Apex Replay Debugger with Current File successfully ran');
```

Comment:
I verified this on my machine while testing and against production.
The `SFDX: Launch Apex Replay Debugger with Current File` command does not display any notifications...
Are you seeing notifications on your machine?

Lines 130-145 should be removed. and in its place:
```
    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Launch Apex Replay Debugger with Current File', 1);
    await browser.keys(['F5']);
    await utilities.pause(1);

    await browser.keys(['F5']);
    await utilities.pause(1);

    // also do this:
    // switch to Debug Console
    // in the Debug look for "launchRequest: args={"name":"Launch Apex Replay Debugger","type":"apex-replay","request":"launch","
    // and look for |EXECUTION_STARTED
    // and look for |EXECUTION_FINISHED
    // and look for Apex Replay Debugger session terminated.
```